ansible/roles/common/tasks/main.yml

```diff
@@ -10,9 +10,10 @@
     become: yes
 
   - name: Create terminal color file and make it executable
-    file:
-      path: "/etc/profile.d/termcolor.sh"
-      state: touch
+    copy:
```

Comment:
What's the difference?

Reply:
Touch does create the file but also actually touches the file every time which makes subspace report changes when it didn't really change. Copy "" with force: no only does something if the file doesn't exist.

Reply:
Aaaah interesting.

Reply:
Would something like `state: present` work here? I feel like the `file` module in ansible should be able to handle this.

Reply:
There is no present state for file, but I think that's because this whole line of thinking is wrong. The following template task will actually create the file and supports `mode` and other file creation things because that actually makes sense and ansible is good.

Reply:
Aah this makes sense :+1: